### PR TITLE
[hi]Localize content/en/docs/reference/glossary/cla.md

### PR DESCRIPTION
**Pull request by taniaduggal**
Signed-off-by: Tania Duggal <103496926+taniaduggal@users.noreply.github.com>
fixes #35600

### CODE REVIEW

**Comment by bishal7679** [comment]
/retest

__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Tags:** Reusable

**Comment by taniaduggal** [comment]
@anubha-v-ardhan please review :)

File: content/hi/docs/reference/glossary/cla.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+aka: 
+tags:
+- community
+---
+ शर्तें जिसके तहत एक {{< glossary_tooltip text="योगदानकर्ता" term_id="contributor" >}} अपने योगदान के लिए एक ओपन सोर्स प्रोजेक्ट को लाइसेंस देता है।
+
+ 
+<!--more--> 
+
+CLA ओपन सोर्स समुदाय में योगदानकर्ताओं द्वारा योगदान की गई सामग्री और बौद्धिक संपदा (IP) से उत्पन्न कानूनी विवादों को हल करने में सहायक है।
```
Comment (divya-mohan0209) [inline content/hi/docs/reference/glossary/cla.md:19]:
```suggestion
सीएलए योगदान सामग्री और बौद्धिक संपदा से जुड़े कानूनी विवादों को सुलझाने में मदद करते है|
```

**Review by divya-mohan0209 — DISMISSED** [review]
Slight nit.

**Review by divya-mohan0209 — COMMENTED** [review]
/approve

**Comment by NitishKumar06** [comment]
/lgtm

Ready to be merged!

**Pull request merged**